docs/specs/main.tex

```diff
@@ -0,0 +1,185 @@
+\documentclass[10pt]{article}
```

Comment:
As discussed, let's unify method names, change the following names:
send_license_req --> request_license,
get_license_req --> get_license_request,
send_license --> issue_license,
call_nullify_license --> use_license,
req_service --> request_service